src/views/identity-checks-completed/identity-checks-completed.njk

```diff
@@ -0,0 +1,98 @@
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+ {% from "govuk/components/button/macro.njk" import govukButton %}
+
+ {% set errors = pageProperties.errors %}
+
+ {% extends "layouts/default.njk" %}
+
+ {% set title = i18n.identityChecksTitle %}
+ {% block main_content %}
+
+  {% set errorPrefix = "wicc" %}
+  {% if errors[errorPrefix + "-day"] %}
+    {% set errorMessageText = errors[errorPrefix + "-day"].text %}
+  {% elif errors[errorPrefix + "-month"] %}
+    {% set errorMessageText = errors[errorPrefix + "-month"].text %}
+  {% elif errors[errorPrefix + "-year"] %}
+    {% set errorMessageText = errors[errorPrefix + "-year"].text %}
+  {% endif %}
+
+   <div>
+     <form action="" method="POST">
+       {% include "partials/user_name.njk" %}
+
+       <div class="govuk-form-group {% if errors %}govuk-form-group--error{% endif %}">
+         <fieldset class="govuk-fieldset">
+           <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+             {{ i18n.identityChecksTitle }}
+           </legend>
+           {% if i18n.dateOfBirthHint %}
+             <span id="wicc-hint" class="govuk-hint">{{ i18n.dateOfBirthHint }}</span>
+           {% endif %}
+           {% if errors %}
+             <span class="govuk-error-message">
+               <span class="govuk-visually-hidden">Error:</span> {{ errorMessageText }}
+             </span>
+           {% endif %}
+
+           <div class="govuk-date-input" id="wicc">
+             <div class="govuk-date-input__item">
+               <div class="govuk-form-group">
+                 <label class="govuk-label govuk-date-input__label" for="wicc-day">
+                   {{ i18n.day }}
+                 </label>
+                 <input class="govuk-input govuk-date-input__input govuk-input--width-2{% if errors %} govuk-input--error{% endif %}"
+                        id="wicc-day"
+                        name="wicc-day"
+                        value="{{ payload["wicc-day"] | default('') }}">
+               </div>
+             </div>
+             <div class="govuk-date-input__item">
+               <div class="govuk-form-group">
+                 <label class="govuk-label govuk-date-input__label" for="wicc-month">
+                   {{ i18n.month }}
+                 </label>
+                 <select class="govuk-select{% if errors %} govuk-select--error{% endif %}"
```

Comment:
so if we have error in date or year... this will so red?
Even though there is no problems with month.

Reply:
Yes correct.